samples/event-sourcing-esdb-simple/src/main/java/io/eventdriven/ecommerce/core/serialization/EventSerializer.java

```diff
@@ -0,0 +1,58 @@
+package io.eventdriven.ecommerce.core.serialization;
+
+import com.eventstore.dbclient.EventData;
+import com.eventstore.dbclient.EventDataBuilder;
+import com.eventstore.dbclient.ResolvedEvent;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import io.eventdriven.ecommerce.core.events.EventTypeMapper;
+
+import java.io.IOException;
+import java.util.UUID;
+
+public final class EventSerializer {
+  public static final JsonMapper mapper = (JsonMapper) new JsonMapper()
```

Comment:
You can easily use spring ObjectMapper instead of creating new JsonMapper.
Example in my repository:

https://github.com/nowakprojects/EventModelingGitHubClone/blob/workshop/start/backend-kotlin/src/main/kotlin/pl/zycienakodach/eventmodeling/githubclone/shared/infrastructure/JsonEventMapper.kt

Reply:
Updated in https://github.com/oskardudycz/EventSourcing.JVM/pull/1/commits/f064e1892b59d153e7294846e07720a756f1447b 👍